cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,314 @@
+        summary_issue_counts["Issue Probability"].append(false_but_true_count / num_examples)
+    return pd.DataFrame.from_dict(summary_issue_counts).sort_values(
+        by=["Issue Probability"], ascending=False
+    )
+
+
+def rank_classes_by_multilabel_quality(
+    labels=None,
+    pred_probs=None,
+    *,
+    class_names=None,
+    joint=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """
+    Returns a Pandas DataFrame with three overall class label quality scores summarizing all examples annotated with each  class in a multi-label classification dataset.
+    Details about each score are listed below under the Returns parameter. By default, classes are ordered
+    by "Label Quality Score", ascending, so the most problematic classes are reported first in the returned DataFrame.
+
+    Score values are unnormalized and may tend to be very small. What matters is their relative
+    ranking across the classes.
+
+    This method works by providing ``labels``, ``pred_probs`` and an optional Confident Joint.
+
+
+
+    **Parameters**: For parameter info, see the docstring of :py:func:`find_overlapping_classes <cleanlab.dataset.find_overlapping_classes>`.
+
+    Returns
+    -------
+    overall_label_quality : pd.DataFrame
+        Pandas DataFrame with cols "Class Index", "Label Issues", "Inverse Label Issues",
+        "Label Issues", "Inverse Label Noise", "Label Quality Score",
+        with a description of each of these columns below.
+        The length of the DataFrame is ``num_classes`` (one row per class).
+        Noise scores are between 0 and 1, where 0 implies no label issues
+        in the class. The "Label Quality Score" is also between 0 and 1 where 1 implies
+        perfect quality. Columns:
+
+        * *Class Index*: The index of the class in 0, 1, ..., K-1.
+        * *Label Issues*: ``count(given_label = k, true_label != k)``, estimated number of examples in the dataset that are labeled as class k but should have a different label.
+        * *Inverse Label Issues*: ``count(given_label != k, true_label = k)``, estimated number of examples in the dataset that should actually be labeled as class k but have been given another label.
+        * *Label Noise*: ``prob(true_label != k | given_label = k)``, estimated proportion of examples in the dataset that are labeled as class k but should have a different label. For each class k: this is computed by dividing the number of examples with "Label Issues" that were labeled as class k by the total number of examples labeled as class k.
+        * *Inverse Label Noise*: ``prob(given_label != k | true_label = k)``, estimated proportion of examples in the dataset that should actually be labeled as class k but have been given another label.
+        * *Label Quality Score*: ``p(true_label = k | given_label = k)``. This is the proportion of examples with given label k that have been labeled correctly, i.e. ``1 - label_noise``.
+
```

Comment:
the dataframe you returned in your PR description has the wrong order of columns. The order needs to match the existing dataframe for multi-class classification datasets, which I believe is also the order you listed here in the docstring.

![Screen Shot 2023-04-07 at 6 00 59 PM](https://user-images.githubusercontent.com/1390638/230696611-7e61958e-454d-4b5b-8a98-691f32bed350.png)

https://docs.cleanlab.ai/stable/tutorials/dataset_health.html

Reply:
Resolved (I've updated the PR comment)